enterprise/models.py

```diff
@@ -1018,6 +1024,18 @@ class EnterpriseCustomerBrandingConfiguration(TimeStampedModel):
         null=True, blank=True, max_length=255,
         validators=[validate_image_extension, validate_image_size]
     )
+    banner_border_color = models.CharField(
+        null=True,
+        blank=True,
+        max_length=7,
+        validators=[validate_hex_color],
```

Comment:
These model fields in `portal-designer` specify a default for the colors (i.e., #FFFFFF). However, I think it makes more sense to not have default colors and leave it up to the frontend to handle the empty case 👍